site/en/docs/devtools/console/api/index.md

```diff
@@ -3,14 +3,20 @@ layout: "layouts/doc-post.njk"
 title: "Console API reference"
 authors:
   - kaycebasques
+  - sofiayem
 date: 2016-03-21
-#updated: YYYY-MM-DD
+updated: 2022-07-22
 description: "Use the Console API to write messages to the Console."
 ---
 
 Use the Console API to write messages to the Console from your JavaScript. See [Get Started With
-Logging Messages To The Console][1] for an interactive introduction to the topic. See [Console
-Utilities API Reference][2] if you're looking for the convenience methods like `debug(function)` or
+Logging Messages To The Console][1] for an interactive introduction to the topic.
```

Comment:
Are we suppose to capitalize "To"? or "to"?

Reply:
Thanks for catching that. We're using sentence case capitalization for titles: https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings.